ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoActiveContractsSpec.scala

```diff
@@ -262,6 +263,27 @@ private[dao] trait JdbcLedgerDaoActiveContractsSpec
     }
   }
 
+  it should "not set the offset" in {
+    for {
+      (_, t1) <- store(singleCreate)
+      (_, t2) <- store(singleCreate)
+      end <- ledgerDao.lookupLedgerEnd()
+      activeContracts <- ledgerDao.transactionsReader
+        .getActiveContracts(
+          activeAt = end,
+          filter = Map(alice -> Set.empty),
+          verbose = true,
+        )
+        .runWith(Sink.seq)
+
+    } yield {
+      activeContracts should not be empty
+      forAll(activeContracts) { ac =>
+        ac.offset shouldBe empty
```

Comment:
Of course. Now that I see this I realize we'd better have an integration test as well (but not a conformance test).

Reply:
yeah, this doesn't test the full picture. but the existing ledger api test tool tests already check for the offset to be available.